package_linux_bin.sh

```diff
@@ -16,6 +16,21 @@ cd vscode || { echo "'vscode' dir not found"; exit 1; }
 export VSCODE_SKIP_NODE_VERSION_CHECK=1
 export VSCODE_SYSROOT_PREFIX='-glibc-2.17'
 
+if [[ "${VSCODE_ARCH}" == "riscv64" ]]; then
+  export ELECTRON_SKIP_BINARY_DOWNLOAD=1
+  export PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
+  # Official electron doesn't support riscv64, adding the checksums of forked electron here
+  # Electron riscv fork might do multiple builds for a same version,
+  # the tag is in the format of v<version>.riscv<build_number>.
+  # In order to be compatible with upstream release scheme, riscv fork also re-upload the latest
+  # RISC-V release to the upstream release tag(v<version>). 
+  # This might be problematic if there is an old electron zip cached locally.
+  rm -f ~/.cache/electron/*/electron-*-linux-riscv64.zip
+  ELECTRON_VERSION="v$(yarn config get target)"
```

Comment:
It would be better to have specific version number and its checksum.
This way, we know for sure which electron version has been used for a specific VSCodium version.
It allows us to be able to reproduce that version if needed.

Reply:
> It would be better to have specific version number and its checksum. This way, we know for sure which electron version has been used for a specific VSCodium version. It allows us to be able to reproduce that version if needed.

Yes, I agree with you. I could use a specific version like v29.4.2.riscv2 but it would require me to make more PRs in the future to maintain the riscv64 revision so I didn't take this approach on the first try.